### PR TITLE
Fix unselected chunk takes when import project

### PR DESCRIPTION
**Pull request by AnonymousWalker**
Previously, after updating the content entity in the database, the collection's active connection was not updated with the new changes and thus it got out of sync. This PR applies the changes to the connection which takes effect when the workbook is reloaded

This change is [Reviewable](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/725)

### CODE REVIEW

File: jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/viewmodel/AddFilesViewModel.kt [hunk 1]
```diff
@@ -129,7 +129,7 @@ class AddFilesViewModel : ViewModel() {
             }
 
             override fun onRequestUserInput(parameter: ImportCallbackParameter): Single<ImportOptions> {
-                return Single.just(ImportOptions(null))
+                return Single.just(ImportOptions(parameter.options))
```
Comment (AnonymousWalker) [inline jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/viewmodel/AddFilesViewModel.kt:132]: default values: import all

**Review by jsarabia — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/725)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @AnonymousWalker)

___
*[`jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/persistence/repositories/ContentRepository.kt` line 213 at r2](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/725#-NRskifGCD5IS86qraEP:-NRskifGCD5IS86qraEQ:b7wk72u) ([raw file](https://github.com/Bible-Translation-Tools/Orature/blob/b021bfc88ed1fbf8f0b3479d1958cd244834a8af/jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/persistence/repositories/ContentRepository.kt#L213)):*
> ```Kotlin
>                 contentDao.update(entity)
> 
>                 activeConnections.keys.find { it.id == entity.collectionFk }?.let { collection ->
> ```

can you move this to a private function called something like updateContentInActiveConnection?

It would help with readability, as well as refactoring if the implementation for active connections changes.

**Review by AnonymousWalker — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/725)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @jsarabia)

___
*[`jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/persistence/repositories/ContentRepository.kt` line 213 at r2](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/725#-NRskifGCD5IS86qraEP:-NRsqvfvAB8k5Sb8-7cy:b-4w8q7g) ([raw file](https://github.com/Bible-Translation-Tools/Orature/blob/b021bfc88ed1fbf8f0b3479d1958cd244834a8af/jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/persistence/repositories/ContentRepository.kt#L213)):*
<details><summary><i>Previously, jsarabia (Joe) wrote…</i></summary><blockquote>

can you move this to a private function called something like updateContentInActiveConnection?

It would help with readability, as well as refactoring if the implementation for active connections changes.
</blockquote></details>

good catch! Resolved

**Review by jsarabia — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/725#-:-NRtAQ8s1zccAvwLjV1M:bnfp4nl)* status: 0 of 2 files reviewed, 1 unresolved discussion

**Pull request merged**